app/modules/flap/endpoints_flappybird.py

```diff
@@ -0,0 +1,100 @@
+import uuid
+from datetime import UTC, datetime
+
+from fastapi import Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.core import models_core
+from app.core.groups.groups_type import GroupType
+from app.core.module import Module
+from app.dependencies import get_db, is_user_a_member
+from app.modules.flap import cruds_flappybird, models_flappybird, schemas_flappybird
+
+module = Module(
+    root="flappybird",
+    tag="Flappy Bird",
+    default_allowed_groups_ids=[GroupType.student],
+)
+
+
+@module.router.get(
+    "/flappybird/scores",
+    response_model=list[schemas_flappybird.FlappyBirdScoreInDB],
+    status_code=200,
+)
+async def get_flappybird_score(
+    skip: int = 0,
+    limit: int = 10,
+    db: AsyncSession = Depends(get_db),
+):
```

Comment:
Could you add a docstring explaining which scores will be returned?
Something like:
> Return the best score of the skip...limit best players